streampark-console/streampark-console-webapp/src/locales/lang/en/flink/app.ts

```diff
@@ -41,7 +41,6 @@ export default {
   podTemplate: 'Kubernetes Pod Template',
   flinkCluster: 'Flink Cluster',
   yarnQueue: 'Yarn Queue',
-  yarnSessionClusterId: 'Yarn Session ClusterId',
```

Comment:
When adding an existing yarn session cluster, many people may not know what `address` and `Yarn Session Cluster` are. It is recommended to give a hint, for example, `address` is the jobmanager address, such as http://hh-hx199w:35154, `Yarn Session Cluster` Is a value similar to application_1656423003647_19237